mesonbuild/dependencies/configtool.py

```diff
@@ -78,6 +79,32 @@ def _sanitize_version(self, version: str) -> str:
             return m.group(0).rstrip('.')
         return version
 
+    @staticmethod
+    def _split_include_dirs(args: T.Iterable[str]) -> T.Tuple[T.List[IncludeDirs], T.List[str]]:
+        """Split include and other compile args
+
+        And convert the include arguments into an IncludeDirs object
+
+        :param args: An iterable of strings
```

Comment:
Perhaps describe it as `include directories`? Otherwise you are just describing the type annotation :)